content/en/docs/started/installing-kubeflow.md

```diff
@@ -158,6 +158,17 @@ The following table lists <b>active distributions</b> that have <b>had a recent
           1.6.1
         </td>
       </tr>
+      <tr>
+        <td>Kubeflow on QBO</td>
+        <td>QBO</td>
+        <td>QBO Kubernetes Engine Community Edition (QKE CE), QBO Kubernetes Engine Cloud (QKE Cloud)</td>
+        <td>
+          <a href="https://docs.qbo.io/#/ai_and_ml?id=kubeflow">Website</a>
+        </td>
+        <td>
+          v1.8.0 (Linux, Windows WSL2)<sup>[<a href="https://github.com/alexeadem/qbo-ce/blob/main/CHANGELOG.md">Release Notes</a>]</sup>
```

Comment:
@alexeadem Please add your distribution version to the layouts, so you can add yourself to the OWNERs files and control distribution Kubeflow version when you become a Kubeflow member.
You check this example:
- DeployKF Version: https://github.com/kubeflow/website/blob/master/layouts/shortcodes/deploykf/latest-version.html
- Reference in the table: {{% deploykf/latest-version %}}

Reply:
@andreyvelich rebased, added qbo to the new table, added OWNERs and reference in table. Let me know if that works. Thanks